### PR TITLE
Data Release for 2024-01-10

### PR DESCRIPTION
**Pull request by dwnoble**
Will fold in new places build once it's completed.

### CODE REVIEW

File: internal/server/v1/propertyvalues/golden/bulk_property_values_in/typeOf.json
Comment (pradh) [inline internal/server/v1/propertyvalues/golden/bulk_property_values_in/typeOf.json:6693]: @n-h-diaz any ideas why this happens?

File: internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json
Comment (pradh) [inline internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json:9992]: @ajaits @hareesh-ms This was from Commerce_NTIA, did we drop these SVs?

**Review by pradh — APPROVED** [review]
Barring a couple of questions, generally LGTM.

Please wait for approval from @ajaits and @jehangiramjad

File: internal/server/v1/page/golden/place_page/asm.Demographics.json
Comment (jehangiramjad) [inline internal/server/v1/page/golden/place_page/asm.Demographics.json:183]: These are coming from some OECD import (not WDI). I am not sure if this is expected?

**Review by jehangiramjad — APPROVED** [review]
Thanks! WDI diffs look good to me. There is one change I saw which is coming from the OECD import and I am not sure whether that's expected or not.

**Comment by dwnoble** [comment]
Updated place bt table. No new changes to the golden file diff with that update.

File: internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json
Comment (ajaits) [reply internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json:9992]: Yes, this is expected as `armedForcesStatus: dcid:Civilian` was added to convert the SVs to Count_Person_15OrMoreYears_**Civilian**_*

**Pull request merged**
